src/main/java/net/apnic/rdap/rdap/RDAPError.java

```diff
@@ -0,0 +1,98 @@
+package net.apnic.rdap.rdap;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class represent an error response in the RDAP protocol.
+ *
+ * @see https://tools.ietf.org/html/rfc7483
+ */
+public class RDAPError
+    extends RDAPObject
+{
+    private List<String> description = null;
```

Comment:
How do you mean receive this from the client. This is sent from the server in error cases.

Need some more clarification on what you mean here.